website/client/src/components/groups/group.vue

```diff
@@ -584,7 +584,6 @@ export default {
         this.$root.$emit('bv::show::modal', 'create-party-modal');
         return;
       }
-
       if (this.isParty) {
```

Comment:
@thewilloftheshadow should this if block get fixed as well? Parties can be group plans as well

Reply:
Ooo I didn't know there were parties that had group plans, yea I'll fix that too